packages/cipherstash-proxy/src/postgresql/backend.rs

```diff
@@ -91,6 +91,8 @@ where
         let sent: u64 = bytes.len() as u64;
         counter!(SERVER_BYTES_RECEIVED_TOTAL).increment(sent);
 
+        self.warn_if_encrypted_value_with_no_config(&code, &bytes)?;
```

Comment:
Handling the check here means double-handling for encrypted statements as we iterate and parse the data again. 

An encrypted statement will only contain an unencrypted column if the configuration for one column was removed `after` a record was encrypted. 

I think the most likely situation for customers is that they accidentally remove the config for the whole table, rather than a single column. In this case, we could display the warning just for `passthrough` statements.